src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ChainedMethodCallWrapCheck.java

```diff
@@ -0,0 +1,448 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2022 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace;
+
+import java.util.regex.Pattern;
+
+import com.puppycrawl.tools.checkstyle.StatelessCheck;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.FullIdent;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
+
+/**
+ * <p>
+ * Checks whether method calls specified by identifier pattern are wrapped.
+ * Wrapping means displaying the contents of the current line on the next line.
+ * It is as simple as adding a line break on the current line and continuing
+ * it on the next line.
+ * </p>
+ * <p>
+ * Rationale: Long chained method calls are difficult to read and comprehend and sometimes
+ * require horizontal scrolling to read the complete method call chain. Wrapping them onto
+ * the next line/s enhances readability.
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code identifierPattern} - Specify method calls to be checked.
+ * By default, no method calls are checked.
```

Comment:
Suggestion: "By default, `identifierPattern`matches nothing, so no method calls are checked. This property must be set in order to use the check."

Reply:
Done.